cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java

```diff
@@ -99,8 +101,14 @@ public static void main(String[] args) {
         registerEventHandlers(slang);
 
 //        long time = System.currentTimeMillis();
+        List<RuntimeException> exceptions = new ArrayList<>();
         try {
-            SlangBuildResults buildResults = slangBuilder.buildSlangContent(projectPath, contentPath, testsPath, testSuites, runTestsInParallel);
+            SlangBuildResults buildResults = slangBuilder.buildSlangContent(projectPath, contentPath, testsPath,
+                    testSuites, shouldValidateDescription, runTestsInParallel);
+            exceptions.addAll(ListUtils.emptyIfNull(buildResults.getCompilationExceptions()));
```

Comment:
maybe we should have a convention not to set null value for this field to avoid checks - WDYT?